src/main/java/io/github/toomanybugs1/QuickFarm/Main.java

```diff
@@ -24,151 +24,159 @@
-                Ageable newBlockAge = (Ageable) b.getBlockData();  // ... and here "b" would be replaced by "block"
-                newBlockAge.setAge(0);
-
-                // Update the player's inventory to reflect the use of the seed during auto-replanting.
-                for (int i = 0; i < player.getInventory().getSize(); i++) {
-                    ItemStack itm = player.getInventory().getItem(i);
-                    if (itm != null && itm.getType().equals(seed.getType())) {  // Find the item we just planted,
-                        itm.setAmount(itm.getAmount() - 1);  // decrement the item's amount
-                        player.getInventory().setItem(i, itm.getAmount() > 0 ? itm : null);  // Remove item if amt == 0
-                        player.updateInventory();  // update the player's inventory
-                        break;  // Once found and updated, no need to continue looping through the inventory.
-                    }
-                }
-
-                // mcmmo should only reward xp if the crop is fully grown
-                if (mcmmo != null && age.getAge() == age.getMaximumAge()) {
-                    McMMOPlayer mcPlayer = UserManager.getPlayer(player);
-                    ExperienceAPI.addXpFromBlockBySkill(block.getState(), mcPlayer, PrimarySkillType.HERBALISM);
-                }
-            }
-        }
-    }
-
-    /**
-     * Returns the plantable version of the given block, if one exists.
-     * 
-     * @param block The block of which to get the plantable version.
-     * @return The plantable version of the given block if it exists, otherwise null.
-     */
-    private ItemStack getPlantableSeed(Block block) {
-        // Get the seed corresponding to the block just broken.
-        switch(block.getType()) {
-        case BEETROOTS:
-            return new ItemStack(Material.BEETROOT_SEEDS);
-        case CARROTS:
-            return new ItemStack(Material.CARROT);
-        case POTATOES:
-            return new ItemStack(Material.POTATO);
-        case WHEAT:
-            return new ItemStack(Material.WHEAT_SEEDS);
-            // case NEW_FARM_PLANT:
-            // return new ItemStack(Material.NEW_FARM_SEED);
-        default:  // Indicate no corresponding seed if "block" wasn't a valid crop.
-            return null;
-        }
-    }
+	List<String> enabledPlayers;
+	mcMMO mcmmo;
+
+	@Override
+	public void onEnable() {
+
+		Bukkit.getPluginManager().registerEvents(this, this);
+
+		this.saveDefaultConfig();
+
+		this.enabledPlayers = this.getConfig().getStringList("players-enabled");
+
+		this.mcmmo = (mcMMO) Bukkit.getServer().getPluginManager().getPlugin("mcMMO");
+
+		if (this.mcmmo == null)
+			getLogger().info("This server does not have mcMMO. Disabling mcMMO features.");
+		else
+			getLogger().info("mcMMO has been detected!");
+	}
+
+	@Override
+	public boolean onCommand(@NotNull CommandSender sender, Command cmd, @NotNull String label, String[] args) {
```

Comment:
The annotation is not really required here as it has no noteable effect and not everyone has the library to begin with (yes, I'm a filthy Eclipse User)

Reply:
I just changed it to reduce the number of annoying warnings